### ISSUE TITLE
whedon is reporting on missing references

### ISSUE THREAD

**Issue by dvberkel** [drwaseemsheikh/audiometry#1]
@drwaseemsheikh I am reviewing your article [Audiometry: A Model-View-ViewModel (MVVM) Application Framework for Hearing Impairment Diagnosis](https://github.com/openjournals/joss-reviews/issues/2016#)

and whedon is reporting on [missing references](https://github.com/openjournals/joss-reviews/issues/2016#issuecomment-574915935). Could you take a look if whedon is missing something, or that it is correctly reported on the missing references.

Thanks.

**Comment by waseemsheikh** [comment]
The references were compiled using BIBTEX and I gathered all the necessary documentation for each paper using Google Scholar BIBTEX entry and sometimes there are some fields missing in the BIBTEX entry but I don't think that those fields are critical for citation. At least, I don't know how to get that information besides using Google Scholar BIBTEX citation and I followed that.

**Comment by waseemsheikh** [comment]
It's complaining on the missing DOIs but not all papers have their DOIs available. At least, I couldn't find all DOIs for the references. However, all the other information is in the correct BIBTEX format and complete.

Waseem

**Comment by drwaseemsheikh** [comment]
The waseemsheikh account is also mine which I used to reply to your comments. Please let me know if anything else is needed from my side. Thanks!

**Comment by drwaseemsheikh** [comment]
The references were compiled using BIBTEX and I gathered all the necessary documentation for each paper using Google Scholar BIBTEX entry and sometimes there are some fields missing in the BIBTEX entry but I don't think that those fields are critical for citation. At least, I don't know how to get that information besides using Google Scholar BIBTEX citation and I followed that.

It's complaining on the missing DOIs but not all papers have their DOIs available. At least, I couldn't find all DOIs for the references. However, all the other information is in the correct BIBTEX format and complete.

Waseem
